getting_started/topics/first-boot/distribution-files-product.adoc

```diff
@@ -3,14 +3,16 @@
 
 The {project_name} Server is contained in one distribution file:
 
-* 'RH-SSO-{project_version}.[zip|tar.gz]'
-
-The 'RH-SSO-{project_version}.[zip|tar.gz]' file is the server-only distribution. It contains only the scripts and binaries
-to run the {project_name} server.
-
-To unpack these files run the `unzip` or `gunzip` and `tar` utilities.
-
+* 'rh-sso-{project_version}.GA.zip'
 
+NOTE: This file can be downloaded in the Red Hat customer portal
+https://access.redhat.com/downloads[Downloads] section, within the "JBoss
```

Comment:
Would it not be better to link to https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso

Reply:
Done